Source/AGXUnreal/Private/Terrain/AGX_MovableTerrainComponent.cpp

```diff
@@ -0,0 +1,923 @@
+#include "Terrain/AGX_MovableTerrainComponent.h"
+#include "Terrain/AGX_ShovelComponent.h"
+#include "Shapes/HeightFieldShapeBarrier.h"
+#include "Shapes/AGX_ShapeComponent.h"
+#include "Materials/AGX_ShapeMaterial.h"
+#include "Materials/AGX_TerrainMaterial.h"
+#include "Utilities/AGX_StringUtilities.h"
+#include "Utilities/AGX_NotificationUtilities.h"
+#include "AGX_InternalDelegateAccessor.h"
+#include "AGX_RigidBodyComponent.h"
+#include "AGX_LogCategory.h"
+#include "AGX_Simulation.h"
+#include "TimerManager.h"
+#include "Engine/SimpleConstructionScript.h"
+#include "Engine/SCS_Node.h"
+#include "NiagaraComponent.h"
+#include "NiagaraDataInterfaceArrayFunctionLibrary.h"
+#include "NiagaraFunctionLibrary.h"
```

Comment:
We are kind of neat about includes, and the copyright text.
You can go through your changes and make sure you have the copyright text at the top of each file (see any other file for reference), both in .cpp and .h files.
And for the includes, they should bee categorized with // AGX Dynamics for Unreal includes  or // Unreal Engine includes and put in alphabetical order, again, see any other file for reference. This may seems overly neat, but it actually really helps with readability and finding issues.
Another good rule of thumb is that AGXUnreal headers comes first, then UE headers, then Standard library includes, if any.